md_template/model.md

```diff
@@ -19,11 +19,16 @@ from simple_gp import SimpleGP
 m = SimpleGP()
 with open("gp.generated.tex", "w") as f:
     f.write("$%s$" % m.latex(excluded=["models", "units"]))
+
+# TODO: this doesn't work! 
```

Comment:
see https://github.com/hoburg/gpkit/pull/900 for the fix, which hopefully @whoburg gets around to merging soon...